services/softlayer_network_storage.go

```diff
@@ -141,7 +141,7 @@ func (slns *softLayer_Network_Storage_Service) DeleteIscsiVolume(volumeId int, i
 }
 
 func (slns *softLayer_Network_Storage_Service) GetIscsiVolume(volumeId int) (datatypes.SoftLayer_Network_Storage, error) {
-	response, err := slns.client.DoRawHttpRequest(fmt.Sprintf("%s/%d/getObject.json", slns.GetName(), volumeId), "GET", new(bytes.Buffer))
+	response, err := slns.client.DoRawHttpRequestWithObjectMask(fmt.Sprintf("%s/%d/getObject.json", slns.GetName(), volumeId), []string{"serviceResourceBackendIpAddress", "lunId"}, "GET", new(bytes.Buffer))
```

Comment:
@maximilien 

The first SL service without object mask will not retrieve `serviceResourceBackendIpAddress `,`lunId ` as defaults. Here is a sample result:
`{"accountId":278444,"capacityGb":500,"createDate":"2015-06-20T21:20:19+08:00","guestId":null,"hardwareId":null,"hostId":null,"id":5453539,"nasType":"ISCSI","serviceProviderId":1,"storageTypeId":"5","upgradableFlag":true,"username":"IBM01SL278444-336","serviceResourceName":"PerfStor Aggr aggr_staassjc0101_hp01"}`

If we want to fetch information including `serviceResourceBackendIpAddress ` and `lunId `, we need to call SL API with object mask.

`{"accountId":278444,"capacityGb":500,"createDate":"2015-06-20T21:20:19+08:00","guestId":null,"hardwareId":null,"hostId":null,"id":5453539,"nasType":"ISCSI","serviceProviderId":1,"storageTypeId":"5","upgradableFlag":true,"username":"IBM01SL278444-336","lunId":"101","serviceResourceBackendIpAddress":"10.1.206.78","serviceResourceName":"PerfStor Aggr aggr_staassjc0101_hp01"}`